### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by preciousbetine**
Added a .gitignore file
Setup Linters
Added project description in README file
Created index.html and style.css files
Ignored some more files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+
+If you like this project, give it a star!
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (mckent05) [inline README.md:53]: - Good job, please use the new [Microverse Readme](https://github.com/microverseinc/readme-template), do not delete any of the readme's header or body, and remember to customise the content of the readme, even the logo to a more personalized one 😄

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (mckent05) [inline test.md:1]: - According to the project requirements, this file should be ignored by git, but it is still visible in your repo, please fix this 🙏

File: test.md [hunk 3]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
Comment (Ibizugbe) [inline test.md:1]: As requested by the previous reviewer, it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that the test.md file should be ignored by git😉.
testgitignore 
the file is still present on GitHub which is an indication that the file hasn't been ignored by git. kindly fix this issue🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @preciousbetine 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d


 ## Highlights
- [ ] The right github-flow method has been used.
- [ ] Project has been professional.
- [x] Commit messages are meaningful and descriptive.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] I think your pull request description will look more professional if it has a general summary before the bullet points or highlights, you can do well to consider that in your subsequent pull request description🤗.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 3 above)
Comment (preciousbetine) [reply test.md]: I have added the file to the gitignore. I don't know why it is still showing 🤔

**Review by iambenkis — APPROVED** [review]
### Hi  @preciousbetine  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


cr-approve

## Highlights

GitHub review feature :heavy_check_mark: 
Correct git flow :heavy_check_mark: 
Linters  :heavy_check_mark: 

## Optional 
**_Every comment with the [**OPTIONAL**] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**